code/datums/diseases/chronic_illness.dm

```diff
@@ -0,0 +1,103 @@
+				to_chat(affected_mob, span_danger("You feel a buzzing in the brain."))
+				SEND_SOUND(affected_mob, sound('sound/weapons/flash_ring.ogg'))
+			if(SPT_PROB(0.5, seconds_per_tick))
+				affected_mob.adjustBruteLoss(1)
+			if(SPT_PROB(0.5, seconds_per_tick))
+				switch(rand(1,3))
+					if(1)
+						scramble_dna(affected_mob, 1, 0, 0, rand(15,45))
+					if(2)
+						scramble_dna(affected_mob, 0, 1, 0, rand(15,45))
+					if(3)
+						scramble_dna(affected_mob, 0, 0, 1, rand(15,45))
+		if(4)
+			if(prob(30))
+				affected_mob.playsound_local(affected_mob, 'sound/effects/singlebeat.ogg', 100, FALSE, use_reverb = FALSE)
+			if(SPT_PROB(1, seconds_per_tick))
+				to_chat(affected_mob, span_danger("You feel an grusome pain in your chest!"))
+				if(prob(75))
+					affected_mob.vomit(45,TRUE)
+			if(SPT_PROB(1, seconds_per_tick))
+				affected_mob.adjustStaminaLoss(100, FALSE)
+				affected_mob.visible_message(span_warning("[affected_mob] collapses!"))
+				if(prob(30))
+					to_chat(affected_mob, span_danger("Your vision blurs as you faint!"))
+					affected_mob.AdjustSleeping(10)
+			if(SPT_PROB(0.5, seconds_per_tick))
+				to_chat(affected_mob, span_danger("[pick("You feel as your atoms accelerate in place.", "You feel like you're being torn apart!")]"))
+				affected_mob.emote("scream")
+				affected_mob.adjustBruteLoss(10)
+			if(SPT_PROB(1, seconds_per_tick))
+				switch(rand(1,3))
+					if(1)
+						scramble_dna(affected_mob, 1, 0, 0, rand(15,45))
+					if(2)
+						scramble_dna(affected_mob, 0, 1, 0, rand(15,45))
+					if(3)
+						scramble_dna(affected_mob, 0, 0, 1, rand(15,45))
+		if(5)
+			switch(pick(1,2,3,4))
+				if(1)
+					to_chat(affected_mob, span_notice("You feel your atoms begin to realign. You're safe. For now."))
+					src.stage = 1
+				if(2)
+					to_chat(affected_mob, span_boldwarning("There is no place for you in this timeline."))
+					affected_mob.adjustStaminaLoss(100, forced = TRUE)
+					playsound(affected_mob.loc, 'sound/magic/repulse.ogg', 100, FALSE)
+					affected_mob.emote("scream")
+					for(var/mob/living/viewers in viewers(3, affected_mob.loc))
+						viewers.flash_act()
+					new /obj/effect/decal/cleanable/plasma(affected_mob.loc)
+					affected_mob.visible_message(span_warning("[affected_mob] is erased from the timeline!"), span_userdanger("You are ripped from the timeline!"))
+					qdel(affected_mob)
+				if(3)
+					affected_mob.visible_message(span_warning("[affected_mob] is torn apart!"), span_userdanger("Your atoms accelerate into criticallity!"))
+					affected_mob.gib()
+				if(4)
+					if(affected_mob.stat == CONSCIOUS)
+						affected_mob.visible_message(span_danger("[affected_mob] clutches at [affected_mob.p_their()] chest as if [affected_mob.p_their()] heart is stopping!"), \
+						span_userdanger("You feel a horrible pain as your heart phases out of your chest!"))
+						var/obj/item/organ/internal/heart/heart = affected_mob.get_organ_slot(ORGAN_SLOT_HEART)
+						qdel(heart)
+						src.stage = 1
```

Comment:
is it intentional that you can dodge the heart ripping/stage resetting by being unconscious